CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.13.0 (2023-03-08)
+
+### New features
+
+- Add better compile-time error messages to diagnose one-shot senders being used as multi-shot senders. ([#586](https://github.com/pika-org/pika/pull/586))
+- Optimize the MPI polling implementation. ([#593](https://github.com/pika-org/pika/pull/593))
```

Comment:
This is unlikely to make it, but we can add still add it separately if it makes it before Wednesday.